src/layout/NestedPicker.js

```diff
@@ -13,6 +13,12 @@ let unflattenObjectBy = _.curry((iteratee, x) =>
 
 let isField = x => x.typeDefault
 
+const div = ({ children, onClick, disabled }) => (
```

Comment:
You should call this `ItemDefault`.

Reply:
Changed. In that case heads up: https://github.com/smartprocure/contexture-react/blob/master/src/exampleTypes/ResultPager.js#L8, since I was following its example.